Documentation/op-guide/v2-migration.md

```diff
@@ -28,6 +28,12 @@ Sometimes an etcd cluster will possibly have v3 data which should not be overwri
 ETCDCTL_API=3 etcdctl get "" --from-key --keys-only --limit 1 | wc -l
 ```
 
+In case v2 data was migrated with TTLs, ensure that stores are consistent by checking the hashes:
+
+```sh
+ETCDCTL_API=3 etcdctl endpoint hash --cluster
```

Comment:
Given that the failure mode for inconsistent data is "member is completely broken", I would recommend increasing the warnings given here.

"You must ensure all members have consistent data before migrating, otherwise some members will not be able to correctly serve queries for new data."

This really should also explain what to do if someone has frequently expiring keys - for instance, someone with keys expiring every second will have an extremely high chance of having inconsistent data.  Should they keep retrying?